js/gallery/src/Gallery.svelte

```diff
@@ -180,11 +180,7 @@
 	<Empty unpadded_box={true} size="large"><Image /></Empty>
 {:else}
 	{#if selected_image !== null && allow_preview}
-		<div
-			on:keydown={on_keydown}
-			class="preview"
-			class:fixed-height={height !== "auto"}
```

Comment:
Got rid of `fixed-height` so that the preview would cover the block and not look like this:

![image](https://github.com/gradio-app/gradio/assets/41651716/81f5a6ba-8453-452c-946c-e3a9906b7059)